CHANGELOG.md

```diff
@@ -0,0 +1,97 @@
+# Changelog
+
+## [Unreleased]
+
+## [v0.1.5] - 2021-01-05
+
+### Added
+
+- Allow support for semicolumn and tabulation in CSV files
+- Collect current distance from OR-Tools
+- Allow initial routes when clustering methods are required
+- Position field for activities
+- Sheduling and initial routes are now compatible
+- Unfeasible missions are removed from routes [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- Log reasons of unfeasible missions [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Read and log OR-Tools version
+- Route costs [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+- Log of matrix computation [#52](https://github.com/Mapotempo/optimizer-api/pull/52)
+- Allow multidepot within clustering [#56](https://github.com/Mapotempo/optimizer-api/pull/56)
+- Introduce direct shipments, pickup and delivery must be consecutive [#51](https://github.com/Mapotempo/optimizer-api/pull/51)
+- Return total_time and total_travel_distance from scheduling heuristic [#63](https://github.com/Mapotempo/optimizer-api/pull/63)
+- Return original ids for missions and vehicles [#73](https://github.com/Mapotempo/optimizer-api/pull/73)
```

Comment:
Only in JSON format

Reply:
CSV format is really specific. I assume that not mentioning it suppose that it is not impacted